README.rst

```diff
@@ -57,56 +42,43 @@ Here is a quick example of plotting an AIA image:
 
 .. code:: python
 
-    >>> import sunpy.map
-    >>> from sunpy.data.sample import AIA_171_IMAGE
-    >>> aia = sunpy.map.Map(AIA_171_IMAGE)
-    >>> aia.peek()
+   >>> import sunpy.map
+   >>> from sunpy.data.sample import AIA_171_IMAGE
+   >>>
+   >>> import matplotlib.pyplot as plt
+   >>>
+   >>> aia = sunpy.map.Map(AIA_171_IMAGE)
+   >>>
+   >>> aia.plot()
+   >>>
+   >>> plt.show()
 
 Getting Help
 ============
 
-For more information or to ask questions about SunPy, check out:
-
--  `SunPy Documentation`_
--  `SunPy Element Channel`_
--  `SunPy Mailing List`_
+For more information or to ask questions about ``sunpy`` or any other SunPy library, check out:
 
-.. _SunPy Documentation: https://docs.sunpy.org/en/stable/
-.. _SunPy Element Channel: https://app.element.io/#/room/#sunpy:openastronomy.org
-.. _SunPy Mailing List: https://groups.google.com/forum/#!forum/sunpy
+-  `sunpy documentation <https://docs.sunpy.org/en/stable/>`__
+-  `SunPy Chat`_
+-  `SunPy mailing list <https://groups.google.com/forum/#!forum/sunpy>`__
 
-Acknowledging or Citing sunpy
-=============================
+Acknowledging or Citing ``sunpy``
+=================================
 
-If you use `sunpy` in your scientific work, we would appreciate citing it in your publications.
-The continued growth and development of `sunpy` is dependent on the community being aware of `sunpy`.
+If you use ``sunpy`` in your scientific work, we would appreciate citing it in your publications.
+The continued growth and development of ``sunpy`` is dependent on the community being aware of ``sunpy``.
 
 Please see https://sunpy.org/about#acknowledging-or-citing-sunpy on how to do this.
```

Comment:
```suggestion
If you use ``sunpy`` in your scientific work, we would appreciate your `citing it in your publications <https://sunpy.org/about#acknowledging-or-citing-sunpy>`__.
The continued growth and development of ``sunpy`` is dependent on the community being aware of ``sunpy``.
```
The last sentence is largely filler, and it'd be nice to get rid of the only bare URL.